app/src/main/java/com/example/composetemplate/presentation/screens/entry_screens/register/AuthViewModel.kt

```diff
@@ -89,20 +93,34 @@ class AuthViewModel(
                         if (success){
                             _signupData.value = signupData.copy(authError = false)
                             uiAuthState.value = UIState.Success(true)
+                            successCallback(true,null)
                         }else {
                             uiAuthState.value = UIState.Error(Constants.UNKNOWN_EXCEPTION)
```

Comment:
It is really unknown exception?

Reply:
the call returning boolean if the response code in the 200 zone.. if its not success I think its 401. there is extension you did to handle it in the client networking so its should work autumatcly no ? and if its not 401 then yes its enknown exception